erts/doc/references/erl_cmd.md

```diff
@@ -252,14 +252,13 @@ described in the corresponding application documentation.
   In this example, an Erlang runtime system is started with environment variable
   `DISPLAY` set to `gin:0`.
 
-- **`-epmd_module Module` (init flag)** - Configures the module responsible to
```

Comment:
Sidenote: I think neither of these is actually "(init flag)" at this point, because they are not interpreted by `init`.